automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -76,14 +76,12 @@ set +e
 
 if [[ "${device_type,,}" == "x86" ]]
 then
-    deviceType="X86"
     flank_template="$FLANK_CONF_X86"
 else
-    deviceType="Arm"
     flank_template="$FLANK_CONF_ARM"
 fi
 
-APK_APP="./samples/${component}/build/outputs/apk/geckoNightly${deviceType}/debug/samples-browser-geckoNightly-${deviceType,,}-debug.apk"
+APK_APP="./samples/${component}/build/outputs/apk/geckoNightlyUniversal/debug/samples-browser-geckoNightly-universal-debug.apk"
```

Comment:
I think you'll need to add `universal` to the list of device types somewhere in the gradle config

Reply:
I am not sure what config that is?

Reply:
Err, looks like we already have `universal` as an architecture, never mind :)
https://github.com/mozilla-mobile/android-components/blob/master/samples/browser/build.gradle#L62-L64